pyo3-ffi-check/macro/src/lib.rs

```diff
@@ -130,16 +130,23 @@ pub fn for_all_fields(input: proc_macro::TokenStream) -> proc_macro::TokenStream
     let mut output = TokenStream::new();
 
     for el in html.select(&selector) {
-        let id = el
+        let field_name = el
             .value()
             .id()
             .unwrap()
             .strip_prefix("structfield.")
             .unwrap();
 
-        let field_ident = Ident::new(id, Span::call_site());
+        let field_ident = Ident::new(field_name, Span::call_site());
+        let bindgen_field_ident = if struct_name == "PyObject" && field_name == "ob_refcnt" {
+            // PyObject since 3.12 implements ob_refcnt as a union; bindgen creates
```

Comment:
This is likely not the time to make that change, but why scrape this instead of using Rustdoc's JSON output? Is that still nightly-only?

Reply:
Looks like it's still unstable, https://github.com/rust-lang/rust/issues/76578

However it would be very nice to use this when available!